runtime/src/lib.rs

```diff
@@ -1581,3 +1582,135 @@ construct_runtime!(
         DistributionWorkingGroup: working_group::<Instance9>::{Pallet, Call, Storage, Event<T>},
     }
 );
+
+// Proxy into the Balances pallet, used to modify behavior of free_balance()
+// MUST ONLY BE USED BY THE STAKING PALLET
```

Comment:
A longer explanation of _why_ this is being added makes sense, also,nice if this all could be in a different file, this file is getting out of hand :D.